pkg/aaparser/aaparser_test.go

```diff
@@ -71,3 +86,33 @@ Copyright 2009-2012 Canonical Ltd.
 		}
 	}
 }
+
+func requiresRoot(t *testing.T) {
+	if !rootEnabled {
+		t.Skip("skipping test that requires root")
+		return
+	}
+	require.Equal(t, 0, os.Getuid(), "This test must be run as root.")
+}
+
+func TestGetVersion(t *testing.T) {
```

Comment:
This is not actually testing the code. A better test would call `parseVersion` with various test cases and check the reutrn values.